scripts/prepare_env.sh

```diff
@@ -6,6 +6,9 @@ set -e
 set -x
 set -o pipefail
 
+graphscope_home="$( cd "$(dirname "$0")/.." >/dev/null 2>&1 ; pwd -P )"
+__version__=`cat ${graphscope_home}/VERSION`
```

Comment:
Weird... I think using `version=xxx` is ok.. why do we need a `__version__`.
and use $(command) instead of `command`